src/platforms/mixpanel/index.ts

```diff
@@ -0,0 +1,23 @@
+import { client } from '@/platforms/mixpanel/client';
+import boxIcon from '@/platforms/mixpanel/icons/box';
+import fullIcon from '@/platforms/mixpanel/icons/full';
+import { auth, platform } from '@/sdk';
+
+export default platform('mixpanel', {
+  auth: [auth.basic({})],
```

Comment:
```suggestion
  auth: auth.basic(),
```
Should we do this?

Reply:
Works with me - this was just want was easiest for me to do, but it's not too much extra effort to change 👍🏻